items/items-perm.md

```diff
@@ -79,15 +87,29 @@ Whether (true/false) the group is the owner of this item. Implies the maximum le
 
 ## Aggregation of permissions from multiple sources
 
-We call *aggregation* the merge of the multiple granted permissions (from possibily different groups) to one permission tuple (indexed by group, item).
+We call *aggregation* the merge of multiple granted permissions to one permission tuple `(group, item)` representing the permissions one group has on an item.
+Those granted permissions may have initially been given to different groups (the group itself or any of its ancestors).
+
+In addition, those granted permissions may have been given from multiple `source groups`,
+and come from different `origin`.
+See details in the [section on database table permissions_granted](#granted-permissions-table-permissions_granted).
+
+For each permission attribute but *can_make_session_official* and *can_enter_\** (so *can_view*, *can_grant_view*, *can_watch*, *can_edit*, and *is_owner*), we take its **maximum level** among all values. In addition, "is_owner=true" makes every other attribute get its maximum possible value.
```

Comment:
`can_request_help` is missing

Reply:
fixed, also added the details for `can_make_session_official` and `can_enter_from` as they are computed in `permissionsView`